### PR TITLE
VACMS-5193 Content Build Separation work

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

See #5193. 

## Testing done

Automated tests and manual tests

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by indytechcook** [comment]
There is an issue with the install repos script.  PR to fix: https://github.com/department-of-veterans-affairs/content-build/pull/94

**Review by acrollet — COMMENTED** [review]
I'm thinking it would be good to update the comments referencing `va-gov/web` in `va_gov_build_trigger/src/WebBuildCommandBuilder.php`

**Comment by indytechcook** [comment]
This will fail build until a new release is cut on the content-build repo and pulled in.

**Comment by ndouglas** [comment]
Oops, sorry @indytechcook -- did my merge mess you up?

**Comment by indytechcook** [comment]
> Oops, sorry @indytechcook -- did my merge mess you up?

No, you are good

**Review by acrollet — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -150,7 +150,7 @@
         "symfony/finder": "^3",
         "symfony/phpunit-bridge": "^5.1",
         "traviscarden/behat-table-comparison": "~0.2",
-        "va-gov/web": "^0.1",
+        "va-gov/content-build": "^0.0.34",
```
Comment (indytechcook) [inline composer.json:153]: Before final merge, do one more `composer req va-gov/content-build`

Comment (ElijahLynn) [reply composer.json:153]: We still need to do this though right, even though it was merged already?

**Pull request merged**